socorro/unittest/processor/test_breakpad_transform_rules.py

```diff
@@ -362,32 +362,29 @@ def test_external_fails_2(self, mocked_subprocess_module):
         processor_meta = get_basic_processor_meta()
 
         mocked_subprocess_handle = mocked_subprocess_module.Popen.return_value
-        mocked_subprocess_handle.stdout.read.return_value = int
+        # This data will fail in json.loads and throw an error
+        mocked_subprocess_handle.stdout.read.return_value = '{ff'
```

Comment:
Previously, this would mock a file type's `read` method to return the `int` type. That'll never happen plus it broke my logging, so I changed it to a string that's not valid JSON.